Loading module dependency 'compiler/gnu/7.3'. 
If you need MPI please load 'mpi/openmpi/2.1-gnu-4.8'. 
Traceback (most recent call last):
  File "convert_petsc_matrix_to_npz.py", line 40, in <module>
    petsc4py.init(sys.argv)
  File "/home/fr/fr_fr/fr_jg1080/.local/lib/python3.6/site-packages/petsc4py/__init__.py", line 42, in init
    PETSc = petsc4py.lib.ImportPETSc(arch)
  File "/home/fr/fr_fr/fr_jg1080/.local/lib/python3.6/site-packages/petsc4py/lib/__init__.py", line 29, in ImportPETSc
    return Import('petsc4py', 'PETSc', path, arch)
  File "/home/fr/fr_fr/fr_jg1080/.local/lib/python3.6/site-packages/petsc4py/lib/__init__.py", line 73, in Import
    module = import_module(pkg, name, path, arch)
  File "/home/fr/fr_fr/fr_jg1080/.local/lib/python3.6/site-packages/petsc4py/lib/__init__.py", line 58, in import_module
    with f: return imp.load_module(fullname, f, fn, info)
  File "/opt/bwhpc/common/devel/python/3.6.5.20180618/lib/python3.6/imp.py", line 243, in load_module
    return load_dynamic(name, filename, file)
  File "/opt/bwhpc/common/devel/python/3.6.5.20180618/lib/python3.6/imp.py", line 343, in load_dynamic
    return _load(spec)
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 658, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 571, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 922, in create_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
ImportError: libpetsc.so.3.12: cannot open shared object file: No such file or directory
